prod-devel-rcar-virtio.yaml

```diff
@@ -336,7 +336,7 @@ components:
     sources:
       - type: repo
         url: https://github.com/xen-troops/android_manifest.git
-        rev: "493c9a919949262095c598cbb6c5cd0258dfc58f" # android-14-xenvm-trout-main
+        rev: "android-15-xenvm-trout-main"
```

Comment:
First, this branch doesn't exist on the repo right now.
And even when it is created, this doesn't look good, considering the commit message.
If moving to Android 15 is still a work in progress, it should be done on a separate branch, and the commit should be changed when the upgrade is considered complete.

Reply:
Done